File: November2022/Java/Maximum69Number.java
// @saorav1994
class Solution {
    public int maximum69Number (int num) {
        String str = String.valueOf(num);
        char [] arr = str.toCharArray();
        int l = str.length();
        int firstSix = -1;
        int lastNine = l;
        for (int i = 0; i < l; i++) {
            if (arr[i] == '6') {
                arr[i] = '9';
                break;
            }
        }
        return Integer.parseInt(String.valueOf(arr));
    }
}

File: November2022/C++/maximum-69-number.cpp
// @Sarang661
class Solution {
public:
    int maximum69Number (int num) {
       string curr = to_string(num);

       for(int index = 0; index < curr.size(); index++){
           if(curr[index] == '6'){
               curr[index] = '9';
               break;
           }
       }

       return stoi(curr);
    }
};